packages/documentation/copy/en/reference/Decorators.md

```diff
@@ -457,14 +459,14 @@ function required(target: Object, propertyKey: string | symbol, parameterIndex:
   Reflect.defineMetadata( requiredMetadataKey, existingRequiredParameters, target, propertyKey);
 }
 
-function validate(target: any, propertyName: string, descriptor: TypedPropertyDescriptor<Function>) {
+function validate(target: any, propertyName: string, descriptor: TypedPropertyDescriptor<AnyFunction>) {
```

Comment:
Honestly, I would just use `any` here, as that matches the previous code box. These docs will become less and less relevant anyway given ECMAScript decorators are now a part of TypeScript.